v2/swagger.yaml

```diff
@@ -0,0 +1,629 @@
+swagger: '2.0'
+info:
+  title: Analytics Core API
+  version: '0.2'
```

Comment:
Can we update version to **2**

Reply:
I'm not too much worried about internal version field. I'm fine with 2 also.